README.md

```diff
@@ -236,6 +250,9 @@ These are the endpoints you can hit to do things.
     PUT /hubs/:hub_slug/off => {message: "ok"}
     POST /hubs/:hub_slug/activities/:activity_slug => {message: "ok"}
     POST /hubs/:hub_slug/devices/:device_slug/commands/:command_slug => {message: "ok"}
+    POST /hubs/:hub_slug/current_activity/commands/:command_slug => {message: "ok"}
+    POST /hubs/:hub_slug/devices/:device_slug/commands/:command_slug/?repeat=3 => {message: "ok"}
```

Comment:
these shouldn't have slashes at the end:

    POST /hubs/:hub_slug/devices/:device_slug/commands/:command_slug?repeat=3 => {message: "ok"}